Gibe.Umbraco.Blog/Models/HardCodedBlogSettings.cs

```diff
@@ -17,6 +17,7 @@ public HardCodedBlogSettings(ISettingsService settingsService)
 		public string IndexName => "ExternalIndex";
 
 		public string BlogPostDocumentTypeAlias => _settingsService.GetSetting("Gibe.Umbraco.Blog.BlogPostDocumentTypeAlias", "blogPost");
+		public string BlogSectionDocumentTypeAlias => _settingsService.GetSetting("Gibe.Umbraco.Blog.BlogSectionDocumentTypeAlias", "blogSection");
 
 		public string UserPickerPropertyEditorAlias => _settingsService.GetSetting("Gibe.Umbraco.Blog.UserPickerPropertyEditorAlias", "Umbraco.UserPicker");
```

Comment:
Nitpick: Seems strange this class is called "HardCodedBlogSettings" and then doesn't use hard-coded values - these don't seem very hard-coded to me!  Should the class be renamed for framework?  Maybe `GibeSettingsBlogSettings`?

Don't have hugely strong opinions either way on this though.